docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -0,0 +1,87 @@
+================
+RPM package
+================
+
+Users can install Trino using the RPM Package Manager (RPM) on Linux
+distributions that support RPM.
```

Comment:
on *some* distributions.

i don't think our RPM will work on all Linux distros using RPM packages, does it?